lib/src/embedded/compilation_dispatcher.dart

```diff
@@ -223,6 +224,11 @@ final class CompilationDispatcher {
       case InboundMessage_CompileRequest_Importer_Importer.notSet:
         _checkNoNonCanonicalScheme(importer);
         return null;
+
+      case InboundMessage_CompileRequest_Importer_Importer.nodePackageImporter:
+        var entryPointUrl =
+            Uri.parse(importer.nodePackageImporter.entryPointPath);
```

Comment:
Again, you _cannot safely parse a path as a URL_. Use `p.toUri()` instead.

Reply:
Updated in b686cc134b845344403352014736bf48806f8734